Tasks/DeployVisualStudioTestAgent/make.json

```diff
@@ -8,7 +8,7 @@
         ],
         "files": [
             {
-                "url": "https://testexecution.blob.core.windows.net/testexecution/3840309/TestExecution.zip",
+                "url": "https://testexecution.blob.core.windows.net/testexecution/4311956/TestExecution.zip",
```

Comment:
>"url": "https://testexecution.blob.core.windows.net/testexecution/4311956/TestExecution [](start = 16, length = 87)

[https://github.com/Microsoft/vsts-tasks/blob/master/Tasks/DeployVisualStudioTestAgent/CreateOfflineTestAgent.ps1](https://github.com/Microsoft/vsts-tasks/blob/master/Tasks/DeployVisualStudioTestAgent/CreateOfflineTestAgent.ps1)

Need to update for offline Agent as well